android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/asset/display/AssetDisplayView.java

```diff
@@ -64,6 +64,15 @@ public void setScreenlet(BaseScreenlet screenlet) {
 		this.screenlet = screenlet;
 	}
 
+	@Override
+	public void showFinishOperation(BaseScreenlet screenlet) {
+		container.removeAllViews();
+		container.addView(screenlet, LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.MATCH_PARENT);
```

Comment:
I think if there is an old screenlet in the container, we should remove it first and then add the new one.